src/events/message/index.ts

```diff
@@ -23,46 +26,79 @@ export default class MessageEvent extends Event {
 			return;
 		}
 
-		if (message.attachments.size > 0) {
-			const file = message.attachments.first(); // only take first attachment as, normally, users cannot send more than one attachment
-
-			if (!file?.name || file?.width) {
-				return;
+		const file = message.attachments.find((attachment) => {
+			if (attachment.name == null || attachment.width) {
```

Comment:
```suggestion
			if (attachment && attachment.name === "" || attachment.width) {
```

Reply:
La première condition sert à rien (on est dans une boucle, l'élément ne peut pas être undefined...
Et puis si le nom est littéralement `""` on skip, alors si le nom est `null`, on skip aussi, je vois pas où tu veux en venir. Du coup
```suggestion
			if (attachment.name|| attachment.width) {
```

Reply:
> La première condition sert à rien (on est dans une boucle, l'élément ne peut pas être undefined...
Ca oui, je me suis trompé
J'ai corrigé la suggestion

Reply:
Du coup ta première partie de condition sera toujours fausse

Reply:
Enfin corrigé